### ISSUE TITLE
[Tracking] SignalR Client does not function in UWP apps with Min Version set to Windows 1803

### ISSUE THREAD

**Issue by anurse** [aspnet/SignalR#2406]
Similar to the System.Memory issues we found in https://github.com/aspnet/SignalR/issues/1886, the System.Memory package contains a `lib/uap10.0.16300/_._` file but the UWP runtime doesn't actually include a version of System.Memory. This means that when the **min version** of an app is targeting `17134` (the runtime version for the 1803 build of Windows), you get failures to load `System.Memory`.

The fix for this is happening in corefx, this issue is just tracking the change so we can verify that SignalR works with the updated package. We know it works with the netstandard2.0 build because if you set the **min version** to `16299` (Fall Creators Update), everything works fine.

**Comment by anurse** [comment]
No shiproom approval is needed here since the issue is in corefx (and they will bring it to shiproom)

**Comment by anurse** [comment]
https://github.com/dotnet/corefx/pull/29970 <-- Corefx pr

**Comment by anurse** [comment]
I'll verify this

**Comment by anurse** [comment]
We are now blocked by https://github.com/dotnet/corefx/issues/30276

**Comment by anurse** [comment]
We need to reference the patched version (`4.5.1`) of `System.Threading.Tasks.Extensions` and `System.Runtime.CompilerServices.Unsafe` directly the make sure they are lifted properly.

**Comment by tomzorz** [comment]
Can confirm that the min version as 16299 workaround solves the issue.

**Comment by anurse** [comment]
There will be a fix in 2.1.1, but it will require adding an additional dependency to your UWP app (it's one that's brought in transitively already, but you'll need to pull the version up). When 2.1.1 goes live, I'll update this issue with the details and post an announcement on aspnet/Announcements

**Comment by anurse** [comment]
The necessary packages are live on NuGet. See https://github.com/aspnet/Announcements/issues/306 for details. I'm closing this issue but feel free to post here if there are any questions about how to deploy the fix. If new issues arise when using the updated packages, please file a new issue!
